rapid7_insightvm/komand_rapid7_insightvm/util/endpoints.py

```diff
@@ -491,15 +491,6 @@ def vulnerability_exception(console_url, id):
         """
         return f"{console_url}/api/3/vulnerability_exceptions/{id}"
 
-    @staticmethod
```

Comment:
Are we sure we didn't need this? It's kind of out of scope for this PR.

Reply:
it's exactly the same function as several lines up and when I ran validators on my branch it said to fix this https://github.com/rapid7/insightconnect-plugins/blob/master/rapid7_insightvm/komand_rapid7_insightvm/util/endpoints.py#L475

Reply:
Ok cool. Thanks for the explanation.